app/models/exercise.rb

```diff
@@ -117,9 +117,10 @@ def to_resource_h
     language_resource_h = language.to_embedded_resource_h if language != guide.language
     as_json(only: %i(name layout editor corollary teacher_info manual_evaluation locale
                      choices assistance_rules randomizations tag_list extra_visible goal
-                     free_form_editor_source initial_state final_state))
+                     free_form_editor_source initial_state final_state custom_expectations))
```

Comment:
Why are we using custom_expectations here if we are overriding it later? 
It also includes guide expectations which isn't necessary, right?